event_location_manager/models/event_event.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+# @authors: Alexander Ezquevo <alexander@acysos.com>
+# Copyright (C) 2016  Acysos S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api, _
+from openerp.exceptions import Warning
+from datetime import datetime, timedelta
+
+
+DFORMAT = "%Y-%m-%d %H:%M:%S"
+
+
+class EventTrackLocation(models.Model):
+    _inherit = 'event.track.location'
+
+    def _get_company(self):
+        return self.env.user.company_id
+
+    company_id = fields.Many2one(string='Company', comodel_name='res.company',
+                                 default=_get_company)
+    capacity = fields.Integer(string='Capacity', required=True)
+    reservation_days = fields.One2many(
+        string='Reservation days',
+        comodel_name='event.track.location.reservation',
+        inverse_name='et_location_id')
+
+
+class EventTrackLocationReservation(models.Model):
+    _name = 'event.track.location.reservation'
+
+    @api.depends('day', 'duration')
+    @api.multi
+    def _get_end_date(self):
+        for res in self:
+            if res.day:
+                date = datetime.strptime(res.day, DFORMAT)
```

Comment:
You can use the method in https://github.com/OCA/OCB/blob/8.0/openerp/fields.py#L1296